core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3879,7 +3893,22 @@ public synchronized void doSubmitDescription( StaplerRequest req, StaplerRespons
     @RequirePOST
     public synchronized HttpRedirect doQuietDown() {
         try {
-            return doQuietDown(false,0);
+            return doQuietDown(false, 0, null);
+        } catch (IOException | InterruptedException e) {
+            throw new AssertionError(); // impossible
+        }
+    }
+
+    /**
+     * Quiet down Jenkins - preparation for a restart
+     * Presented for compatibility.
+     *
+     * @param block Block until the system really quiets down and no builds are running
+     * @param timeout If non-zero, only block up to the specified number of milliseconds
+     */
+    public synchronized HttpRedirect doQuietDown(boolean block, int timeout) {
```

Comment:
Probably shouldn't be synchronized, and it might make sense for it to be deprecated

Reply:
Don't wanna break anything. Let it be synchronized.
And I will mark it as deprecated.